### PR TITLE
Add documention for scheduler metrics

### PR DESCRIPTION
**Pull request by mpoeter**
### Scope & Purpose

Backport of #13811

- [x] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)

#### Backports:

- [x] No backports required

### Testing & Verification

- [x] This change is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

File: Documentation/Metrics/arangodb_scheduler_jobs_dequeued.yaml [hunk 1]
```diff
@@ -9,5 +9,15 @@ exposedBy:
   - coordinator
   - dbserver
   - agent
+  - single
 description: |
-  Total number of jobs dequeued.
+  The total number of jobs dequeued from all scheduler queues.
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_scheduler_jobs_dequeued.yaml:14]: I would have suggested to not document the legacy metrics at all, but I have no strong feelings. I would have suggested to remove this YAML file and use `DECLARE_LEGACY_GAUGE` such that the check script does not complain. If we leave it like this we have the strange situation that the manual has a duplication of the documentation but the v2 metrics do not have some of the entries.

File: Documentation/Metrics/arangodb_scheduler_jobs_done.yaml [hunk 2]
```diff
@@ -9,5 +9,12 @@ exposedBy:
   - coordinator
   - dbserver
   - agent
+  - single
 description: |
-  Total number of queue jobs done.
+  The total number of queue jobs done.
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_scheduler_jobs_done.yaml:14]: Same here.

File: Documentation/Metrics/arangodb_scheduler_jobs_submitted.yaml [hunk 3]
```diff
@@ -9,5 +9,12 @@ exposedBy:
   - coordinator
   - dbserver
   - agent
+  - single
 description: |
-  Total number of jobs submitted to the queue.
+  Total number of jobs submitted to the scheduler.
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_scheduler_jobs_submitted.yaml:14]: Same here.

**Review by neunhoef — CHANGES_REQUESTED** [review]
LGTM, but see suggestion in comments. Not completely necessary, but this was so far our policy.

File: Documentation/Metrics/arangodb_scheduler_jobs_dequeued.yaml (hunk 1 above)
Comment (mpoeter) [reply Documentation/Metrics/arangodb_scheduler_jobs_dequeued.yaml:14]: Makes sense!

**Review by neunhoef — APPROVED** [review]
LGTM

**Pull request merged**